x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -436,6 +468,107 @@ export const PolicyList = React.memo(() => {
 
 PolicyList.displayName = 'PolicyList';
 
+const EmptyPolicyTable = React.memo<{
+  loading: boolean;
+  onActionClick: (event: MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => void;
+  actionDisabled: boolean;
+  dataTestSubj: string;
+}>(({ loading, onActionClick, actionDisabled, dataTestSubj }) => {
+  const policySteps = [
+    {
+      title: i18n.translate('xpack.securitySolution.endpoint.policyList.stepOneTitle', {
+        defaultMessage: 'Head over to Ingest Manager.',
+      }),
+      children: (
+        <EuiText color="subdued" size="xs">
+          <FormattedMessage
+            id="xpack.securitySolution.endpoint.policyList.stepOne"
+            defaultMessage="Here, you’ll add the Elastic Endpoint Security Integration to your Agent Configuration."
+          />
+        </EuiText>
+      ),
+    },
+    {
+      title: i18n.translate('xpack.securitySolution.endpoint.policyList.stepTwoTitle', {
+        defaultMessage: 'We’ll create a recommended security policy for you.',
+      }),
+      children: (
+        <EuiText color="subdued" size="xs">
+          <FormattedMessage
+            id="xpack.securitySolution.endpoint.policyList.stepTwo"
+            defaultMessage="You can edit this policy in the “Policies” tab after you’ve added the Elastic Endpoint integration."
+          />
+        </EuiText>
+      ),
+    },
+    {
+      title: i18n.translate('xpack.securitySolution.endpoint.policyList.stepThreeTitle', {
+        defaultMessage: 'Enroll your agents through Fleet.',
+      }),
+      children: (
+        <EuiText color="subdued" size="xs">
+          <FormattedMessage
+            id="xpack.securitySolution.endpoint.policyList.stepThree"
+            defaultMessage="If you haven’t already, enroll your agents through Fleet using the same agent configuration."
+          />
+        </EuiText>
+      ),
+    },
+  ];
+  return (
+    <div data-test-subj={dataTestSubj}>
+      {loading ? (
+        <FormattedMessage
```

Comment:
should this be a [Loader](https://elastic.github.io/eui/#/display/loading) or be shown with a [Progress](https://elastic.github.io/eui/#/display/progress) component?